api/src/pcapi/core/subscription/api.py

```diff
@@ -124,6 +127,16 @@ def activate_beneficiary_for_eligibility(
     return user
 
 
+def check_eligibility_is_applicable_to_user(eligibility: users_models.EligibilityType, user: users_models.User) -> None:
+    is_user_eighteen = user.has_beneficiary_role or (user.age is not None and user.age >= 18)
```

Comment:
Détail : on a une constante qu'il est préférable d'utiliser, même si elle ne changera jamais.
```suggestion
    is_user_eighteen = user.has_beneficiary_role or (user.age is not None and user.age >= users_constants.ELIGIBILITY_AGE_18)
```